clis/teliod/QNAP.md

```diff
@@ -0,0 +1,127 @@
+# QNAP Build
+
+To build Teliod package for QNAP devices there are two alternatives:
+
+- Run cargo build directly:
+```cargo build --verbose --target x86_64-unknown-linux-musl --package teliod --features qnap``` 
+
+- Run build script [Recommended]:
+```../../ci/build_libtelio.py build qnap x86_64 [--debug]```
+
+The build script has an additional stage at the end of the build where it creates the QNAP package.
+
+## REST API
+
+This REST API allows interaction with the Teliod daemon. It provides endpoints for managing the daemon, updating its configuration, and retrieving logs and status information.
+
+### Endpoints
+
+#### 1. **Start the Daemon**
+- **Endpoint**: `/`
+- **Method**: `POST`
+- **Description**: Starts the Teliod Daemon in the background.
+- **Request Body**: None
```

Comment:
Why are we not sending configuration together with start?

Reply:
configuration is being read from file. User should pass configuration values if he wish so?

Reply:
It probably would be nice, but will see, maybe I will change this a bit then implementing ui